.gitignore

```diff
@@ -7,4 +7,7 @@ Gemfile.lock
 *.code-workspace
 _vendor/*
 info.txt
-tmp/
\ No newline at end of file
+tmp/
+_pythonModules/*
```

Comment:
Do we need to include _pythonModules and _RModules?  or not b/c they are old and thus re-pulled during every PR from the easybuild repo???

Reply:
They are no longer part of this repo - they get pulled from the easybuild repo during the build process. 
It's not so relevant anymore but when I would troubleshoot builds on the shell runner I would do 'git status' and it would show me hundreds of files from these two directories because they were not in .gitignore and not in git either. 
That's why I added them to .gitignore.